autoscale-core/src/main/java/com/hpe/caf/autoscale/core/Governor.java

```diff
@@ -45,12 +46,28 @@
      */
     ScalingAction govern(String serviceRef, ScalingAction action, int currentMemoryLimitStage);
 
+    /**
+     *
+     * @param serviceRef the named reference to the service
+     * @return True or False based on if the governor was able to make room for the service
+     * throws ScalerException when orchestrator is unable to scale down another service to make room or if there is no other service that
```

Comment:
- `@throws`
- From the description I don't really understand the difference between a `ScalerException` being thrown and `false` being returned.

Reply:
ScalerException is thrown from the service attempting to be scaled down by the governor

Reply:
Scaler Tread no longer thrown from here